mozdownload/scraper.py

```diff
@@ -242,6 +243,51 @@ def detect_platform(self):
         else:
             return "%s%d" % (mozinfo.os, mozinfo.bits)
 
+    def get_checksum_url(self):
+        """Get the checksum url for current version."""
+        filename = 'SHA512SUMS'
+        url = os.path.split(os.path.split(os.path.split(self.url)[0])[0])[0] + '/' + filename
+        return url
```

Comment:
Hm, where do you see such a checksum file on archive.mozilla.org? I have a hard time finding one.